interp/src/interpreter/interpret_group.rs

```diff
@@ -135,13 +135,35 @@ impl WorkingEnvironment {
         }
         self.backing_env
     }
+
+    // For debugging purpose
+    // fn _dump_state(&self, cell: &ir::Cell) {
+    //     println!("{} on cycle {}: ", cell.name(), self.backing_env.clk);
+    //     for p in &cell.ports {
+    //         let p_ref: &ir::Port = &p.borrow();
+    //         println!("  {} : {}", p_ref.name, self.get_as_val(p_ref).as_u64());
+    //     }
+    //     match self
+    //         .backing_env
+    //         .cell_prim_map
+    //         .get(&(cell as *const ir::Cell))
+    //         .unwrap()
+    //     {
+    //         &Primitive::StdReg(ref reg) => {
+    //             println!("  internal state: {}", reg.val)
+    //         }
+    //         &Primitive::StdMemD1(ref mem) => {
+    //             println!("  memval : {}", mem.data[0])
+    //         }
+    //         _ => {}
+    //     }
+    // }
```

Comment:
We should probably either commit to leaving the debugging code or remove the comment